features/step_definitions/assertion_steps.rb

```diff
@@ -0,0 +1,11 @@
+Then("I should see {string}") do |content|
+  expect(page).to have_content content
+end
+
+Then("I should be on Login page") do
+  visit new_user_session_path
```

Comment:
please make use of expectations when writing the assertions:
`expect(current_path).to eq new_user_session_path`

Reply:
Changed according to comments.